exist-core/src/main/java/org/exist/xquery/functions/fn/FunCurrentDateTime.java

```diff
@@ -48,11 +48,11 @@ public class FunCurrentDateTime extends Function {
     public final static FunctionSignature fnCurrentDateTime =
         new FunctionSignature(
             new QName("current-dateTime", Function.BUILTIN_FUNCTION_NS),
-            "Returns the xs:dateTimeStamp (with timezone) that is current at some time " +
+            "Returns the xs:dateTime (with timezone) that is current at some time " +
             "during the evaluation of a query or transformation in which " +
             "fn:current-dateTime() is executed.",
             null,
-            new FunctionReturnSequenceType(Type.DATE_TIME_STAMP,
+            new FunctionReturnSequenceType(Type.DATE_TIME,
```

Comment:
According to the spec for [fn:current-dateTime()](https://www.w3.org/TR/xpath-functions-31/#func-current-dateTime), this function is supposed to return `xs:dateTimeStamp`, so this change should be undone.

Reply:
@joewiz: Thank you for pointing this out.  I undid this change.